installers/connect-installer/release_notes.md

```diff
@@ -1,18 +1,10 @@
 # ftrack Connect installer release Notes
 
-## v24.3.0rc3
-2024-03-27
+## v24.4.0
+2024-04-02
 
 * [changed] Release connect 3.0.0rc3
```

Comment:
Perhaps merge these two lines to:

* [new] Release connect 3.0.0

Reply:
Yes, done.